### ISSUE TITLE
[Feature Request]: Option to hide menu icon in macOS

### ISSUE THREAD

**Issue by gpanders** [Heroic-Games-Launcher/HeroicGamesLauncher#2819]
### Problem description

The Heroic app places an icon in the menu bar with no ability to remove it.

### Feature description

I would like an option to hide the menu icon in macOS.

https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/issues/2017 made a similar request, but was closed without any discussion. To answer the question "why?" in that issue: because space in the menu bar on macOS is a limited resource, and the user should have the ability to choose which icons are present. Even macOS itself lets users add or remove system icons from the menu.

The menu icon does not provide any value to me (the Heroic app is present in the dock, so if I need to open it I can click there), so I'd like to remove the icon to make space for other icons.

Thanks!

### Alternatives

As far as I can tell there are no alternatives.

**Comment by drewkerr** [comment]
Agreed. It's unnecessary and at the very least needs to be optional.
